fcl/dunefdvd/gen/background/prodbackground_radiological_decay0_dunevd10kt_1x8x6.fcl

```diff
@@ -1,71 +1,41 @@
 #include "services_dune.fcl"
-#include "dune_radiological_model_decay0_vd_1x8x6.fcl"
+#include "workflow_radiological_decay0_dunevd10kt.fcl"
 
-process_name: MARLEYGen
+process_name: RadioGen
 
 services:
 {
-   @table::dunefd_services
+   #  @table::dunefd_services
    TFileService:          { fileName: "prodradiological_hist.root" }
    TimeTracker:           {}
+  MemoryTracker:         {} # default is one
    RandomNumberGenerator: {}                 # ART native random number generator
    FileCatalogMetadata:  @local::art_file_catalog_mc
    message:              @local::dune_message_services_prod
+  @table::dunefd_services
+  IFDH: {}
+  @table::dunefdvd_1x8x6_3view_30deg_services
 }
 
 source:
 {
    module_type: EmptyEvent
    timestampPlugin: { plugin_type: "GeneratedEventTimestamp" }
    maxEvents:   10          # Number of events to create
-   firstRun:    20000047    # Run number to use for this file
+   firstRun:    20000047           # Run number to use for this file
```

Comment:
Was this a typo?

Reply:
What, the first run number? It was already there and I have no clue why would anyone choose this number.

Reply:
Actually I mean the indentation change of the comment.

Reply:
I see now. Correcting it